travis/manylinux1_x86_64-glpk/build_lpsolve.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+LPS_VER="5.5.2.5"
+
+# Must download this manually due to old OpenSSL
+# wget https://sourceforge.net/projects/lpsolve/files/lpsolve/${LPS_VER}/lp_solve_${LPS_VER}_source.tar.gz
```

Comment:
I assume this is no longer true?

Reply:
Yes, you have to do this locally. `wget` is broken on this ancient image. I think it's due to sourceforge using a redirect or something. I left the command for reference purposes.